vent/api/plugins.py

```diff
@@ -483,6 +483,14 @@ def builder(self,
         except Exception as e:  # pragma: no cover
             self.logger.error("unable to change to directory: " + str(cwd) +
                               " because: " + str(e))
+        # remove untagged images
+        deleted_dict = self.d_client.images.prune({"dangling":"true"})
```

Comment:
this will remove all images that are untagged, but we only want to remove images that are untagged that were created by vent.  fortunately images created by vent have labels on them, so you should be able to query the images by the label and then remove the ones that don't have a tag.